mkdocs.yml

```diff
@@ -30,7 +30,7 @@ nav:
       - Query: generated/api/query_api.md
       - Transformation Functions: generated/api/transformation_functions_api.md
       - ValidationReport: generated/api/validation_report_api.md
-      - Job configuration: generated/api/job_configuration.md
+      - Job: generated/api/job.md
```

Comment:
I don't understand why you created a new template but then don't use the old one anymore?

Reply:
Because I included the job configuration in the job page now

Reply:
ah okay there was no template for the job config.